Cloudflare/zerotrust_dns/ingest/parser.yml

```
@@ -0,0 +1,154 @@
+name: zerotrust-dns
```

To be coherent with other parsers for Cloudflare
```suggestion
name: Clouflare DNS Gateway
```